RecoVertex/BeamSpotProducer/src/PVFitter.cc

```diff
@@ -87,6 +86,8 @@ void PVFitter::initialize(const edm::ParameterSet& iConfig,
   errorScale_        = iConfig.getParameter<edm::ParameterSet>("PVFitter").getUntrackedParameter<double>("errorScale");
   sigmaCut_          = iConfig.getParameter<edm::ParameterSet>("PVFitter").getUntrackedParameter<double>("nSigmaCut");
   fFitPerBunchCrossing=iConfig.getParameter<edm::ParameterSet>("PVFitter").getUntrackedParameter<bool>("FitPerBunchCrossing");
+  useOnlyFirstPV_    = iConfig.getParameter<edm::ParameterSet>("PVFitter").getUntrackedParameter<bool>("useOnlyFirstPV");
+  minSumPt_          = iConfig.getParameter<edm::ParameterSet>("PVFitter").getUntrackedParameter<double>("minSumPt");
```

Comment:
Thank you Francesco.
All parameters which may affect the event (i.e reconstructed data, in this case) must be tracked. Untracked parameters are those which do not affect the event, e.g. histogram ranges, debug keys, etc. In this module basically all parameters were defined untracked (not only the new ones added by you): if the beamspot fitted here is the one going to be used in reco, then all relevant parameters which enter in its definition must get tracked (not only the new ones added by you)